glambie/data/submission_system_interface.py

```diff
@@ -0,0 +1,125 @@
+"""
+Interface to the GlaMBIE data submission system.
+
+The submission system standardises the data recieved by GlaMBIE in
+a number of different ways and presents the output within a Google Bucket.
+
+This module provides access functions for that bucket.
+"""
+
+from io import BytesIO
+import json
+import os
+import re
+from typing import List, Optional
+
+from google.cloud import storage
+import pandas as pd
+
+from glambie.const.data_groups import GlambieDataGroup
+from glambie.const.regions import RGIRegion
+
+_storage_client = storage.Client(project='glambie')
+
+_SUBMISSIONS_BUCKET_URI = 'gs://glambie-submissions'
+SUBMISSION_SYSTEM_FLAG = 'glambie_submission_system'
```

Comment:
Having seen the rest of the code, the naming of this still confuses me

Reply:
The problem is that the DataCatalogue and Timeseries class design assumes that they're based around files on disk. For the submission system, this isn't the case. However, rather than doing a large refactor, I've introduced this as a placeholder meaning "This catalogue came from the submission system, not the local filesystem".

I've explained this a bit more, and named it `SUBMISSION_SYSTEM_BASEPATH_PLACEHOLDER` which is hopefully more direct.

Reply:
Oh, I see. That makes more sense now. Thanks.